docs/site/content/docs/latest/tanzu-debugging-tips.md

```diff
@@ -101,3 +95,24 @@ management cluster.
     docker stop 2f505b8b0c8a && docker rm 2f505b8b0c8a
     docker stop 3dd82b26ac04 && docker rm 3dd82b26ac04
     ```
+## Aditional helpful commands for debugging clusters
```

Comment:
Will need a blank line between the closing codeblock and this header.

Reply:
done